src/main/java/seedu/exercise/logic/parser/ParserUtil.java

```diff
@@ -29,6 +29,10 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String ADD_CATEGORY_EXERCISE = "exercise";
+    public static final String ADD_CATEGORY_REGIME = "regime";
+    public static final String SUGGEST_TYPE_BASIC = "basic";
+    public static final String SUGGEST_TYPE_POSSIBLE = "possible";
 
```

Comment:
Maybe you can place these constants in the command themselves instead of `ParserUtil` as they are specific to the commands and the Util classes do not need to know them?

Reply:
These constants are used in ParserUtil, and the individual Parser classes (AddCommandParser and SuggestCommandParser). 
Is it good practice for ParserUtil to use constants declared in the specific command classes?

Reply:
Then maybe another alternative is to place them in the AddCommandParser and SuggestCommandParser. I'm okay with it being in ParserUtil also. Best place for constants is in the class that will actually need them and makes the most sense to be scoped in so we can reduce coupling

Reply:
Ok. I will store them in the respective Parser classes for now. We can discuss this again